docs/reference/node-feature-client-reference.md

```diff
@@ -63,3 +63,23 @@ The `--registry-password-stdin` flag enables reading of registry password from s
 #### --registry-token-stdin
 
 The `--registry-token-stdin` flag enables reading of registry token from stdin.
+
+## export
+
+Commands for exporting node features locally.
```

Comment:
nit:
```suggestion
Commands for exporting node features to the terminal.
```

Reply:
That's slightly imprecise imo, I used "stdout or a file" instead. WDYT?